README.md

```diff
@@ -31,68 +32,76 @@
        transcribe  Transcribe one or more audio file(s)
    ```
 
+
 ## Example usage
 
 - A normal real time session using a .wav file as the input audio
 
    ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ speechmatics transcribe--url $URL --lang en --ssl-mode=none example_audio.wav
+   $ URL=ws://realtimeappliance.yourcompany:9000/v2
+   $ speechmatics transcribe --url $URL --lang en --ssl-mode none example_audio.wav
    ```
 
-- A normal real time session with a locally running container
+- Show the messages that are going over the websocket connection
 
    ```shell
-   $ URL=ws://127.0.0.1:9000/v2
-   $ speechmatics transcribe --url $URL --lang en --ssl-mode=none example_audio.wav
+   $ speechmatics transcribe -v --url $URL --lang en --ssl-mode none example_audio.wav
    ```
 
-- Show the messages that are going over the websocket connection
+- The CLI also accepts an audio stream on standard input; transcribe the piped input audio
 
    ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ speechmatics transcribe -v --url $URL --lang en --ssl-mode=none example_audio.wav
+   $ cat example_audio.wav | speechmatics transcribe --ssl-mode none --url $URL --lang en -
    ```
 
-- Similar to the first example, but this time the input audio is piped in
+- Pipe audio directly from the microphone (example uses MacOS with [ffmpeg](https://ffmpeg.org/ffmpeg-devices.html#avfoundation))
 
-   ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ cat example_audio.wav | speechmatics transcribe --ssl-mode=none --url $URL --lang en -
-   ```
+    First list available input devices with
 
-- The CLI also accepts an audio stream on standard input, meaning that you can stream in a live microphone feed for example.
+    ```shell
+    $ ffmpeg -f avfoundation -list_devices true -i ""
+    ```
 
-   **MacOS example with ffmpeg**
+    There needs to be at least one available microphone attached to your computer.
```

Comment:
perhaps enumerate those notes for easier reading?

Reply:
I think it's readable in the current state: https://github.com/speechmatics/speechmatics-python/tree/update-docs#example-usage

Reply:
This is quite a long block; first sentence relates to the previous paragraph while second describes the command which follows a few lines below.

It may be a personal bias; for me it would be clearer to either separate those bits or reorganize it so that explanation is always *before* or *after*. Up to you though.